### ISSUE TITLE
Flutter - atlasUrl is missing in config 

### ISSUE THREAD

**Issue by kunalgharate** [mongodb-university/realm-template-apps#168]
Atlas url is used in project but is missing in config

```
class Config extends ChangeNotifier {
  late String appId;
  late String? atlasUrl;
  late Uri baseUrl;

  Config._create(dynamic realmConfig) {
    appId = realmConfig['appId'];
    atlasUrl = realmConfig['dataExplorerLink'];
    baseUrl = Uri.parse(realmConfig['baseUrl']);
  }

  static Future<Config> getConfig(String jsonConfigPath) async {
    dynamic realmConfig =
        json.decode(await rootBundle.loadString(jsonConfigPath));

    var config = Config._create(realmConfig);

    return config;
  }
}

```

**Comment by krollins-mdb** [comment]
Sorry for the slow response, @kunalgharate. What method did you use to get the template app? (clone the repo, App Services UI, App Services CLI)

I should definitely have added a placeholder in the `atlasConfig.json` file for cloning the repo. If you got the template app from App Services (UI or CLI), the generated config file should have the url. If not, I'll need to investigate the pipeline.

**Comment by dacharyc** [comment]
Hey folks - I have just merged PR #182 which should resolve this issue. I'm going to go ahead and close this issue, but if you still have a problem, feel free to re-open, @kunalgharate .
